cmd/common/frontends/frontends.go

```diff
@@ -12,16 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package audit
+package frontends
 
-import (
-	"github.com/spf13/cobra"
+import "context"
 
-	commonaudit "github.com/inspektor-gadget/inspektor-gadget/cmd/common/audit"
-)
-
-func NewAuditCmd() *cobra.Command {
-	cmd := commonaudit.NewCommonAuditCmd()
-	cmd.AddCommand(newSeccompCmd())
-	return cmd
+type Frontend interface {
+	Output(payload string)
+	Error(severity int, message string)
```

Comment:
`Error(severity logger.Level, message string)`  maybe? The compilation is failing for me:

``` 
=> => # *Frontend does not implement frontends.Frontend (wrong type for Error method)                                                                                                                                             
 => => # have Error(severity logrus.Level, message string)                                                                                                                                                                         
 => => # want Error(severity int, message string)
```

Reply:
Fixed, missed to commit that file's update ;)